papers/ian_kenney/ian_kenney.rst

```diff
@@ -109,13 +109,14 @@ Software has become increasingly essential to research. In many areas, it underl
 
 However, simply sharing code is not sufficient to fulfill FAIR guidelines. In fact, making software FAIR compliant requires significant investment and often expert knowledge on the part of the developers, especially if the code was written specifically for a particular research project. For example, the Python ecosystem is so dynamic that it is common for research code to rapidly become obsolete or unusable if a new version of a key library is released. To fulfill the Reusability tenet of FAIR alone, code should include documentation, version control, and dependency management. Ideally, it would also include unit tests, examples, and packaging. Even when code is released in reference to a publication, it often falls short of ideal FAIR standards. A short survey of publications in Scopus :cite:`noauthor_scopus_nodate` and the Journal of Open Source Software :cite:`noauthor_journal_nodate` over 2017–-2021 identified that out of a total 720 papers citing MDAnalysis :cite:`michaud-agrawal_mdanalysis_2011, gowers_mdanalysis_2016`, only 43 linked to code available on a version control platform such as GitHub, GitLab, or Bitbucket. Of these, only 18 met the requirements of best practices: they implemented unit tests, comprehensive documentation, and some means of installation.
 
 Two major factors contribute to the lack of open-source FAIR compliant code. Firstly, code is typically written by scientists with no formal training or support in programming, for whom implementing FAIR principles can pose an intimidating and tedious barrier. Secondly, despite the substantial investment of effort and time required to implement best practices, publishing FAIR software is not typically appreciated with academic recognition or reward. Fostering a culture of open-source FAIR software requires addressing both.
 
+.. [#] Based on a search for repositories containing ``import MDAnalysis``.
```

Comment:
```suggestion
.. [#] This figure is based on searching GitHub for repositories containing ``import MDAnalysis``.
```

This might be a bit clearer?